ha_lineairdb.cc

```diff
@@ -880,118 +878,79 @@ int ha_lineairdb::create(const char*, TABLE*, HA_CREATE_INFO*, dd::Table*) {
   return 0;
 }
 
+std::string ha_lineairdb::extract_key() {
+  if (is_primary_key_exists()) return get_key_from_mysql();
+  else return autogenerate_key();
```

Comment:
Is `else` necessary here?

Reply:
I think it is a good naming and abstraction of the processing.

Reply:
`else` is not necessary. However, I thought it makes sense because both are regular behavior, and not "error check and return" paths. What do you think?

Reply:
I understand the intent. :+1:
The compiler would generate the same machine code anyway, and thus I think it's fine, as long as the code formatter or lint tool doesn't complain.

Another idea to indicate that both paths are regular, is using `[[likely]]` for if statement.